components/header/header.hbs

```diff
@@ -0,0 +1,10 @@
+<header class="page__header">
```

Comment:
Seems like this template is crossing concerns.. `page__header` is part of a `page` module.

Reply:
Agree, have removed the page__header from the header template. This should be used in the Page component only.